pkg/streamingpromql/engine_test.go

```diff
@@ -2968,6 +3010,7 @@ func TestCompareVariousMixedMetricsVectorSelectors(t *testing.T) {
 		}
 
 		expressions = append(expressions, fmt.Sprintf(`predict_linear(series{label=~"(%s)"}[1m], 30)`, labelRegex))
+		expressions = append(expressions, fmt.Sprintf(`quantile_over_time(0.5, series{label=~"(%s)"}[1m])`, labelRegex))
```

Comment:
I've used `scalar(series{label="i"})` with other scalar parms in the gauntlet to provide more variation.

```suggestion
		expressions = append(expressions, fmt.Sprintf(`quantile_over_time(scalar(series{label="i"}), series{label=~"(%s)"}[1m])`, labelRegex))
```

Reply:
Changed in f65eaad